Added main / sub animations - Fixed bugs

Arignir opened the pull request:
Added main / sub animations.
Removed DemoScene.
Fixed orientation bug with Sylvanas mesh.

Arignir committed:
Added main / sub animations
Removed DemoScene.
Fixed orientation bug with Sylvanas mesh.

Gaspard-- commented on include/Animations.hpp:
```diff
@@ -0,0 +1,11 @@
+{
+  static constexpr char IDLE[] = "idle";
+  static constexpr char WALK[] = "Walk";
+  static constexpr char STAND[] = "Stand";
```
I think `static` isn't necessary. Yçu are missing `const` however (in C++11 it is implied by `constexpr`, but not in C++14)

Gaspard-- reviewed (changes requested):
Very clean code, minor problem with missing `const`s.

Arignir committed:
Made animations const - nonstatic
Removed g++ -Wno* flags

Arignir commented:
Done ! 

I have also updated the `CMakeLists.txt` and removed the `-Wno`* warnings, as they shoudln't be needed now that i made them system includes.

Can you check that too please ? Thanks.

Gaspard-- commented:
Compiles and runs properly on fedora. I'll trust you for windows & arch.
